integration/presto/src/main/prestodb/org/apache/carbondata/presto/CarbondataModule.java

```diff
@@ -21,6 +21,8 @@
 
 import static java.util.Objects.requireNonNull;
 
+import org.apache.carbondata.core.constants.CarbonCommonConstants;
+import org.apache.carbondata.core.util.CarbonProperties;
```

Comment:
Why this change is required?

Reply:
prestodb compile was broken by #3885 , They have copied code but not import statement. so, fixed it. Mentioned in PR description also

Reply:
ok